apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-content.js

```diff
@@ -66,17 +66,14 @@ function getTourContent() {
 		},
 		{
 			heading: __( 'Undo any mistake', 'full-site-editing' ),
-			description: __(
-				"Simply click the Undo button if you've made a mistake.",
-				'full-site-editing'
-			),
+			description: __( "Click the Undo button if you've made a mistake.", 'full-site-editing' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.